subsys/bluetooth/Kconfig

```diff
@@ -222,6 +222,65 @@ config BT_DATA_LEN_UPDATE
 	  Enable support for Bluetooth v4.2 LE Data Length Update procedure.
 endif # BT_CONN
 
+
+config BT_ISO
+	bool "Bluetooth Isochronous Channel Support [EXPERIMENTAL]"
+	depends on BT_CONN
+	help
+	  Select this to enable Isochronous Channel support.
+
+if BT_ISO
+
+config BT_ISO_MAX_CHAN
+	int "Maximum number of simultaneous ISO channels"
+	depends on BT_ISO
+	default BT_MAX_CONN
+	range 1 64
+	help
+	  Maximum number of simultaneous Bluetooth isochronous channels
+	  supported.
+
+config BT_ISO_TX_BUF_COUNT
+	int "Number of Isochronous TX buffers"
+	default 1
+	range 1 255
+	help
+	  Number of buffers available for outgoing Isochronous channel packets.
+
+config BT_ISO_TX_FRAG_COUNT
+	int "Number of ISO TX fragment buffers"
```

Comment:
I assume this is referring to fragmenting SDUs in the host and not to fragmentation in ISOAL?

Reply:
Correct